trl/trainer/sft_trainer.py

```diff
@@ -445,9 +445,9 @@ def _prepare_dataset(
         dataset,
         tokenizer,
         packing,
-        dataset_text_field,
+        dataset_text_field: str,
```

Comment:
not super important, but maybe we should type the rest of the function signature for consistency?

Reply:
Agree in general, but I didn't want to spend to much time on it since I plan to refactor SFT data preprocessing, to be more aligned with what's done in other trainers (which is clearer IMO)